api-extractor/src/definitions/ApiProperty.ts

```diff
@@ -26,8 +26,8 @@ class ApiProperty extends ApiMember {
   /**
    * {@inheritdoc ApiItem.onResolveReferences }
    */
-  protected onResolveReferences(): void {
-    super.onResolveReferences();
+  protected onCompleteeReferences(): void {
+    super.onCompleteReferences();
 
     if (this.documentation.hasReadOnlyTag) {
```

Comment:
Also, you misspelled `onCompleteeReferences` but the `super` is spelled right.  In TypeScript, this means the method simply won't ever get called without any error.  Seems like we're missing a unit test for this code path.

Eventually \@virtual / \@override enforcement should catch these mistakes.